src/Simulation/EntryPointDriver/Driver.cs

```diff
@@ -112,25 +101,37 @@ private static async Task<int> Simulate<T>(IEntryPoint<T> entryPoint, ParseResul
                     DisplayCustomSimulatorError(simulator);
                     return 1;
                 }
-                await DisplayEntryPointResult(entryPoint, parseResult, createSimulator);
+                await RunSimulator(parseResult, createSimulator);
             }
             return 0;
         }
 
+        /// <summary>
+        /// Submits the entry point to Azure Quantum.
+        /// </summary>
+        /// <param name="parseResult">The command-line parsing result.</param>
+        private async Task Submit(ParseResult parseResult)
+        {
+            // TODO: Use an actual quantum machine.
+            var machine = new SimulatorMachine();
+            var output = await machine.ExecuteAsync(entryPoint.Info, entryPoint.CreateArgument(parseResult));
+            foreach (var (result, frequency) in output.Histogram)
+            {
+                Console.WriteLine($"{result} (frequency = {frequency})");
```

Comment:
I assume that this piece will still change (I think we had said the default would be to not show the entire histogram but instead the most frequent value?).

Reply:
Yes, this was just temporary since the method that gets the most frequent output is not available yet. Added a TODO.